nginx.conf

```diff
@@ -73,6 +73,9 @@ http {
         add_header Strict-Transport-Security "max-age=31536000; includeSubdomains" always;
         add_header X-Content-Type-Options "nosniff" always;
         add_header 'Referrer-Policy' 'strict-origin';
+        # 'sha256-ruKmkK0iwJgE/F4xuzLY3V2OuzVOOJISav7NURhCKsM=' hugo discuss template (https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/disqus.html)
```

Comment:
do you need this comment for discuss template here?

Reply:
Otherwise I need to remember that this hash is related to this script and remove it once we drop disqus

Reply:
I see, this explains sha below